source/ed/fare2ed.cpp

```diff
@@ -73,6 +72,12 @@ int fare2ed(int argc, const char * argv[])
         return 0;
     }
 
+    // Construct logger and signal handling
+    std::string log_comment = "";
+    if (vm.count("log_comment")) { log_comment = vm["log_comment"].as<std::string>(); }
+    navitia::init_app("ed2nav", "DEBUG", vm.count("local_syslog"), log_comment);
```

Comment:
`ed2nav` shouldn't be `fare2ed` ?

Reply:
also, this whole bloc is duplicated all over the place. It would have been nice to wrap this in a function.

Reply:
:see_no_evil: :fireworks:

Reply:
For real, there is a lot of code blocs that is duplicate in binaries...

Reply:
I agree it's not the best :( But it's not an excuse not to do better :D